Formula/u/ucg.rb

```diff
@@ -1,10 +1,23 @@
 class Ucg < Formula
   desc "Tool for searching large bodies of source code (like grep)"
   homepage "https://github.com/gvansickle/ucg"
-  url "https://github.com/gvansickle/ucg/releases/download/0.3.3/universalcodegrep-0.3.3.tar.gz"
-  sha256 "116d832bbc743c7dd469e5e7f1b20addb3b7a08df4b4441d59da3acf221caf2d"
   license "GPL-3.0-or-later"
-  head "https://github.com/gvansickle/ucg.git", branch: "master"
+
+  stable do
+    url "https://github.com/gvansickle/ucg/releases/download/0.3.3/universalcodegrep-0.3.3.tar.gz"
+    sha256 "116d832bbc743c7dd469e5e7f1b20addb3b7a08df4b4441d59da3acf221caf2d"
+
+    # Fix Xcode 9 compilation issue: https://github.com/gvansickle/ucg/issues/118
```

Comment:
Is this just for Xcode 9? Or is it 9+? If the former, we can probably drop it.

Reply:
Should be 9+.